extensions/ql-vscode/src/remote-queries/shared/variant-analysis.ts

```diff
@@ -1,5 +1,92 @@
+import { Repository } from './repository';
+
+export interface VariantAnalysisSubmission {
+  startTime: number,
+  controllerRepoId: number,
+  actionRepoRef: string,
+  query: {
+    name: string,
+    filePath: string,
+    language: VariantAnalysisQueryLanguage,
+
+    // Base64 encoded query pack.
+    pack: string,
```

Comment:
Do we need this to be stored like this? Would it perhaps make more sense to store it as a regular file?

Reply:
This is gone now :)